src/main/java/com/rcv/GuiConfigController.java

```diff
@@ -168,33 +176,42 @@
   private ButtonBar buttonBar;
 
   public void buttonNewConfigClicked() {
-    Logger.log(Level.INFO, "Creating new config.");
-    GuiContext.getInstance().setConfig(null);
-    selectedFile = null;
-    clearConfig();
+    if (checkForSaveAndContinue()) {
+      Logger.log(Level.INFO, "Creating new config.");
+      GuiContext.getInstance().setConfig(null);
+      selectedFile = null;
+      clearConfig();
+    }
   }
 
-  public void buttonLoadConfigClicked() {
-    FileChooser fc = new FileChooser();
-    if (selectedFile == null) {
-      fc.setInitialDirectory(new File(System.getProperty("user.dir")));
-    } else {
-      fc.setInitialDirectory(new File(selectedFile.getParent()));
+  private void loadFile(File loadFile) {
```

Comment:
Feels a little confusing for the function and its only param to share the same name.

Reply:
Renaming this to `fileToLoad` and `saveFile` to `fileToSave`.